packages/graphiql/src/components/DocExplorer.js

```diff
@@ -142,6 +168,24 @@ export class DocExplorer extends React.Component {
 
   // Public API
   showDoc(typeOrField) {
+    if (!typeOrField) { return; }
+
+    if (this.props.onShowDoc) {
+      let typeName;
+      let fieldName;
+      if (isType(typeOrField)) {
+        typeName = typeOrField.name;
+        fieldName = '';
```

Comment:
I think `null` is more appropriate than the empty string.

```suggestion
        fieldName = null;
```